homeassistant/components/wled/__init__.py

```diff
@@ -125,9 +126,11 @@ async def _async_update_data(self) -> WLEDDevice:
             raise UpdateFailed(f"Invalid response from API: {error}") from error
 
 
-class WLEDEntity(CoordinatorEntity):
+class WLEDEntity(CoordinatorEntity[WLEDDevice]):
```

Comment:
```suggestion
class WLEDEntity(CoordinatorEntity):
```
Because you already annotated `coordinator` field as `WLEDDataUpdateCoordinator` this is not used as field annotation overrides it. It's useful when plain `DataUpdateCoordinator` is used.

Reply:
Aha! Thanks.